lib/monitor/monitor_test.go

```diff
@@ -95,21 +69,27 @@ func TestKeyOrder(t *testing.T) {
 
 	dbg.TestOutput(testing.Verbose(), 3)
 	m := make(map[string]string)
-	m["servers"] = "1"
+	m["machines"] = "1"
 	m["hosts"] = "1"
 	m["bf"] = "2"
-	m["rounds"] = "3"
+	// create stats
+	stat := NewStats(m)
+	m1 := NewSingleMeasure("round", 10)
+	m2 := NewSingleMeasure("setup", 5)
+	stat.Update(m1)
+	stat.Update(m2)
+	str := new(bytes.Buffer)
+	stat.WriteHeader(str)
+	stat.WriteValues(str)
+
+	stat2 := NewStats(m)
+	stat2.Update(m1)
+	stat2.Update(m2)
 
-	for i := 0; i < 20; i++ {
-		// First set up monitor listening
-		stat := NewStats(m)
-		NewMonitor(stat)
-		time.Sleep(100 * time.Millisecond)
-		b := bytes.NewBuffer(make([]byte, 1024))
-		stat.WriteHeader(b)
-		dbg.Lvl2("Order:", strings.TrimSpace(b.String()))
-		if strings.Contains(b.String(), "rounds, bf") {
-			t.Fatal("Order of fields is not correct")
-		}
+	str2 := new(bytes.Buffer)
```

Comment:
Sorry, writing too fast. Of course, you're right. I thought it's the code where it tries to connect to the sink.